README.md

```diff
@@ -22,83 +22,95 @@ The client library currently contains a single high-level pattern for synchronou
 RPC clients and servers. Other patterns (e.g. for pub/sub) could be added.
 
 Quick Start
-===========
+-----------
 
-    $ git clone git://github.com/djs55/message-switch
-    $ cd message-switch
-    $ make
+```sh
+git clone git://github.com/djs55/message-switch
+cd message-switch
+make
+```
 
 Run the server (by default it will bind to 127.0.0.1:8080)
 
-    $ ./switch.native 
+```sh
+./switch.native
```

Comment:
`_build/default/bin/message-switch`?

Reply:
Oh yes, I lint-ed the README before the port and forgot to update the paths afterwards..